inc/Engine/Optimization/GoogleFonts/AbstractGFOptimization.php

```diff
@@ -28,6 +28,27 @@ abstract class AbstractGFOptimization extends AbstractOptimization {
 		'optional' => 1,
 	];
 
+	/**
+	 * Flag for whether google fonts have been detected (Default: true)
+	 *
+	 * @since 3.8.8
+	 *
+	 * @var bool
+	 */
+	protected $has_google_fonts;
```

Comment:
```suggestion
	protected $has_google_fonts = true;
```

We can set the default value here, and it will also simplify the related method. Else we declare it like this, the default value is `null`, not a boolean (it would trigger errors in PHPStan for example)

Reply:
Indeed, this was how I had it initially. I changed it during the writing of the tests, but it should be reverted.